salt/apps/ocw/ocw_aka/install.sls

```diff
@@ -0,0 +1,35 @@
+{% set users = [fsuser, ocwuser, sshacs] %}
+{% set root_directory = '/ocwdata/apache-tomcat/webapps/' %}
```

Comment:
Same as above, we should make this reasonable. Possibly just put it to `/var/www/ocw`